sdk/communication/azure-communication-chat/azure/communication/chat/_chat_client.py

```diff
@@ -56,7 +61,7 @@ class ChatClient(object): # pylint: disable=client-accepts-api-version-keyword
     def __init__(
             self,
             endpoint,  # type: str
-            credential,  # type: CommunicationTokenCredential
+            credential: Union[TokenCredential, AzureKeyCredential],
```

Comment:
Chat team, is this change fine here? Method was regenerated with a new signature

Reply:
@LuChen-Microsoft need your confirmation here